trm/setting_wcp_arg.tex

```diff
@@ -0,0 +1,79 @@
+\[
+    \boxed{\text{All constraints in this subsection assume } \done_{i} = 1 }
+\]
+To this end We impose the following constraints
+\begin{description}
+	\def\nRows{\rowOffsetAddress}\item[\underline{Processing row $n^\circ(\nRows)$:}] 
+		we impose that
+		\[
+			\wcpCallToLeq {
+				anchorRow = i - \anchorRow         ,
+				relOffset = \nRows                 ,
+				argOneHi  = \trmAddrHi             ,
+				argOneLo  = \rawAddrLo             ,
+				argTwoHi  = (1 << 4) - 1           ,
+				argTwoLo  = 0                      ,
+			}
+		\]
+		we further impose that
+		\[
+			\resultMustBeTrue {
+				anchorRow = i - \anchorRow      ,
+				relOffset = \nRows              ,
+			}
+		\]
+		\saNote{}
+		The above enforces that the address is written on up to 20 bytes.
+		
+	\def\nRows{\rowOffsetAddressTrm}\item[\underline{Processing row $n^\circ(\nRows)$:}] 
+		we impose that
+		\[
+			\wcpCallToLeq {
+				anchorRow = i - \anchorRow         ,
+				relOffset = \nRows                 ,
+				argOneHi  = 0                      ,
+				argOneLo  = \trmLeadHi             ,
+				argTwoHi  = 0                      ,
+				argTwoLo  = (1 << 12) - 1          ,
```

Comment:
Same comment: `(1 << 12 * 8) - 1`

Reply:
Also given that below you are using `256 ** bla` it may make more sense to use `256 ** bla` everywhere rather than a mixture of `(1 << bla)` and `256 ** bla`.